frontend/src/ts/url.ts

```diff
@@ -62,13 +62,20 @@ export const GETTING_STARTED_LINK_EN = '/getting-started.html';
 export const SUBMMIT_ISSUE_LINK =
   'https://github.com/awslabs/clickstream-analytics-on-aws/issues';
 
+export const getSolutionVersion = () => {
+  const configJSONObj: ConfigType = JSON.parse(
+    localStorage.getItem(PROJECT_CONFIG_JSON) || '{}'
+  );
+  return configJSONObj?.solution_version ?? 'latest';
```

Comment:
The document is using `1.0.x` to match all 1.0 releases, like `1.0.0.`, `1.0.1`, `1.0.2`.

I think we can't directly use the version as a prefix.

Reply:
Fixed. And `1.1.x` to match all 1.1 releases, like `1.1.0`, `v1.1.1`, `1.1.2`.